Missing SimpleGit package after running npm install

After running npm install -g open-in-vcs-cli, the following error is thrown when trying to use the tool in a git repo:
```
> oiv open .
    Error: Cannot find module 'simple-git'
    Require stack:
    - C:\Users\jmorj\AppData\Roaming\npm\node_modules\open-in-vcs-cli\lib\commands\open.js
    - C:\Users\jmorj\AppData\Roaming\npm\node_modules\open-in-vcs-cli\node_modules\@oclif\config\lib\plugin.js
    - C:\Users\jmorj\AppData\Roaming\npm\node_modules\open-in-vcs-cli\node_modules\@oclif\config\lib\config.js
    - C:\Users\jmorj\AppData\Roaming\npm\node_modules\open-in-vcs-cli\node_modules\@oclif\config\lib\index.js
    - C:\Users\jmorj\AppData\Roaming\npm\node_modules\open-in-vcs-cli\node_modules\@oclif\command\lib\command.js
    - C:\Users\jmorj\AppData\Roaming\npm\node_modules\open-in-vcs-cli\node_modules\@oclif\command\lib\index.js
    - C:\Users\jmorj\AppData\Roaming\npm\node_modules\open-in-vcs-cli\bin\run
    Code: MODULE_NOT_FOUND
```

--- a/package.json
+++ b/package.json
@@ -11,6 +11,8 @@
     "@oclif/command": "^1.8.0",
     "@oclif/config": "^1.17.0",
     "@oclif/plugin-help": "^3.2.3",
+    "open": "^8.4.0",
+    "simple-git": "^2.47.0",
     "tslib": "^1.14.1"
   },
   "devDependencies": {
@@ -26,9 +28,7 @@
     "globby": "^10.0.2",
     "mocha": "^5.2.0",
     "nyc": "^14.1.1",
-    "open": "^8.4.0",
     "rimraf": "^3.0.2",
-    "simple-git": "^2.47.0",
     "ts-node": "^8.10.2",
     "typescript": "^3.9.10"
   },

